RecoTauTag/RecoTau/test/runDeepTauIDsOnMiniAOD.py

```diff
@@ -0,0 +1,46 @@
+# produce pat::Tau collection with the new DNN Tau-Ids from miniAOD 12Apr2018_94X_mc2017
+
+import FWCore.ParameterSet.Config as cms
+from FWCore.ParameterSet.VarParsing import VarParsing
+
+options = VarParsing('analysis')
+options.parseArguments()
+
+process = cms.Process('runDeepTauIDsOnMiniAOD')
+process.options = cms.untracked.PSet()
+process.options.wantSummary = cms.untracked.bool(False)
+process.options.allowUnscheduled = cms.untracked.bool(True)
+process.options.numberOfThreads = cms.untracked.uint32(8)
+process.options.numberOfStreams = cms.untracked.uint32(0)
+
+process.load('FWCore.MessageLogger.MessageLogger_cfi')
+process.MessageLogger.cerr.FwkReport.reportEvery = 100
```

Comment:
I propose make frequency of report variable with number of events to be processed, e.g. like this
```
if process.maxEvents.input.value()>10:
     process.MessageLogger.cerr.FwkReport.reportEvery = process.maxEvents.input.value()//10
if process.maxEvents.input.value()>10000 or process.maxEvents.input.value()<0:
     process.MessageLogger.cerr.FwkReport.reportEvery = 1000
```